start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+nix-shell default.nix
```

Comment:
You don't need to specify default.nix as it is the - guess what - *default* 😜

You only need to execute "nix-shell" so I think we can also just drop the start.sh file entirely?